### PR TITLE
Fix updating the Offset property in the Inspector when dragging item in the notation view

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Resolves: #18507

### CODE REVIEW

**Review by Eism — APPROVED** [review]

File: src/inspector/models/general/generalsettingsmodel.cpp [hunk 1]
```diff
@@ -33,8 +33,12 @@ GeneralSettingsModel::GeneralSettingsModel(QObject* parent, IElementRepositorySe
...
     setTitle(qtrc("inspector", "General"));
     setSectionType(InspectorSectionType::SECTION_GENERAL);
-    setPlaybackProxyModel(new PlaybackProxyModel(this, repository));
-    setAppearanceSettingsModel(new AppearanceSettingsModel(this, repository));
+
+    m_playbackProxyModel = new PlaybackProxyModel(this, repository);
+    m_playbackProxyModel->init();
+
+    m_appearanceSettingsModel = new AppearanceSettingsModel(this, repository);
+    m_appearanceSettingsModel->init();
```
Comment (RomanPudashkin) [inline src/inspector/models/general/generalsettingsmodel.cpp:41]: You should make a similar change for BeamSettingsModel

Comment (Eism) [reply src/inspector/models/general/generalsettingsmodel.cpp:41]: RestBeamSettingsModel

**Comment by DmitryArefiev** [comment]
Tested #18507
 on Win10, Mac13, LinuxUbuntu 22.04 - **FIXED**

**Comment by DmitryArefiev** [comment]
(sorry, closed by accident)

File: src/inspector/models/general/generalsettingsmodel.cpp (hunk 1 above)
Comment (RomanPudashkin) [reply src/inspector/models/general/generalsettingsmodel.cpp:41]: @cbjeukendrup please fix this for the master https://github.com/musescore/MuseScore/pull/18531/commits/0423c039fddada315f0f6a79e8bcf9004fbd9d15

Comment (cbjeukendrup) [reply src/inspector/models/general/generalsettingsmodel.cpp:41]: Finally done, sorry for the inconvenience

**Review by RomanPudashkin — APPROVED** [review]

**Pull request merged**